SPEC.md

```diff
@@ -495,6 +495,9 @@ The ACE must provide a Metadata service on the address given to the applications
 
 Clients querying any of these endpoints MUST specify the `Metadata-Flavor: AppContainer` header.
 
+**NOTE**: Implementations can insert an authorization token into AC_METADATA_URL.
```

Comment:
@eyakubovich the point I was trying to make here is
```
`AC_METADATA_URL`
```
for consistency with above/below